corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -447,6 +450,69 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
             }
         },
 
+        columnStyle: function () {
+            let self = this;
+            let mapColumn = "";
+            if (self.showMap && !self.hasNoItems) {
+                mapColumn = "[map] 300px";
+            }
+            return "display: grid;grid-template-columns: [tiles] auto " + mapColumn
+                + " ;grid-template-rows: auto";
+        },
+
+        addAddressPin: function (geocoder, addressMap, headers, model, addressIndex) {
+            let address = model.attributes.data[addressIndex];
+            geocoder.query(address, function (err, data) {
+                if (err === null) {
+                    let popupText = "";
+                    headers.forEach((header, index) => {
+                        if (header) {
+                            popupText += "<b>" + header + ":</b> " + model.attributes.data[index] + "<br>";
+                        }
+                    });
+                    L.marker(data.latlng)
+                        .addTo(addressMap)
+                        .bindPopup(popupText);
+                }
+            });
+        },
+
+        loadMap: function () {
+            let token = initialPageData.get("mapbox_access_token");
+
+            try {
+                let lat = 30;
+                let lon = 15;
+                let zoom = 3;
+                let addressMap = L.map('module-case-list-map').setView([lat, lon], zoom);
+                L.tileLayer('https://api.mapbox.com/styles/v1/{id}/tiles/{z}/{x}/{y}?access_token=' + token, {
```

Comment:
Not blocking, this is good for MVP, but it might be worth extracting this from here and `entries.js` and moving it somewhere like `maps_utils.js`.

Reply:
Do you mean create a new file `corehq/apps/cloudcare/static/cloudcare/js/maps_utils.js` and put it in there?

Reply:
There's a maps utils file in reports - https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/reports/static/reports/js/maps_utils.js

I haven't read through it to thoroughly understand how much of it is specific to reports vs more generic (`initLayers` looked like code that might be generic). I think ideally we'd have any cross-app functionality in a new `hqwebapp/js/maps_utils.js`, and then any web apps functionality (anything that's shared between the case list map and the form entry adress question) in a new `cloudcare/js/maps_utils.js` file.

But I also think this would be better in a followup PR, if it's the last outstanding thing here. (I'm worried about code conflicts between this and other case list functionality in progress, so smaller PRs merged more frequently are better.)

Also noting I left similar feedback for Soltech's geospatial work ehre: https://github.com/dimagi/commcare-hq/pull/33020#discussion_r1210418688